Increasing Volume Capacity leads to SIGSEGV

When trying to increase the storage capacity quota I am getting the following dump:

```
$> limesctl project set PROJECT_ID volumesv2/capacity=50GiB

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x13a8502]

goroutine 1 [running]:
github.com/sapcc/limesctl/internal/core.(*Project).setBaseUnits(0xc000272600, 0xc0003385a0, 0xc0000c04f0, 0x0)
	/Users/USERNAME/go/src/github.com/sapcc/limesctl/internal/core/parse_quotas.go:245 +0x1d2
github.com/sapcc/limesctl/internal/core.ParseRawQuotas(0xc0003385a0, 0x1528dc0, 0xc000272600, 0xc0000d7340, 0x1, 0x1, 0x0, 0x0, 0xc000272600, 0x0)
	/Users/USERNAME/go/src/github.com/sapcc/limesctl/internal/core/parse_quotas.go:139 +0x421
main.main()
	/Users/USERNAME/go/src/github.com/sapcc/limesctl/main.go:262 +0x9bd```

--- a/internal/core/parse_quotas.go
+++ b/internal/core/parse_quotas.go
@@ -194,9 +194,13 @@ func (c *Cluster) setBaseUnits(limesV1 *gophercloud.ServiceClient, ru *resourceU
 
 	for srv, resMap := range *ru {
 		for res := range resMap {
-			srvRes, exists := cluster.Services[srv].Resources[res]
-			if exists {
-				(*ru)[srv][res] = srvRes.ResourceInfo.Unit
+			if cluster.Services[srv] != nil {
+				srvRes, exists := cluster.Services[srv].Resources[res]
+				if exists {
+					(*ru)[srv][res] = srvRes.ResourceInfo.Unit
+				}
+			} else {
+				errors.Handle(fmt.Errorf("service %s for cluster %s could not be found", srv, cluster.ID), "")
 			}
 		}
 	}
@@ -218,9 +222,13 @@ func (d *Domain) setBaseUnits(limesV1 *gophercloud.ServiceClient, ru *resourceUn
 
 	for srv, resMap := range *ru {
 		for res := range resMap {
-			srvRes, exists := domain.Services[srv].Resources[res]
-			if exists {
-				(*ru)[srv][res] = srvRes.ResourceInfo.Unit
+			if domain.Services[srv] != nil {
+				srvRes, exists := domain.Services[srv].Resources[res]
+				if exists {
+					(*ru)[srv][res] = srvRes.ResourceInfo.Unit
+				}
+			} else {
+				errors.Handle(fmt.Errorf("service %s for domain %s could not be found", srv, domain.Name), "")
 			}
 		}
 	}
@@ -242,9 +250,13 @@ func (p *Project) setBaseUnits(limesV1 *gophercloud.ServiceClient, ru *resourceU
 
 	for srv, resMap := range *ru {
 		for res := range resMap {
-			srvRes, exists := project.Services[srv].Resources[res]
-			if exists {
-				(*ru)[srv][res] = srvRes.ResourceInfo.Unit
+			if project.Services[srv] != nil {
+				srvRes, exists := project.Services[srv].Resources[res]
+				if exists {
+					(*ru)[srv][res] = srvRes.ResourceInfo.Unit
+				}
+			} else {
+				errors.Handle(fmt.Errorf("service %s for project %s could not be found", srv, project.Name), "")
 			}
 		}
 	}

